src/seedu/addressbook/AddressBook.java

```diff
@@ -1046,7 +1048,8 @@ private static boolean isPersonNameValid(String name) {
      * @return whether arg is a valid person phone number
      */
     private static boolean isPersonPhoneValid(String phone) {
-        return phone.matches("\\d+");    // phone nonempty sequence of digits
+    	final String NONEMPTYDIGITS = "\\d+";
```

Comment:
Please check java naming convention to see how to separate two words in naming of final strings. It is good to rename regex both for readability and better maintenance later.